### PR TITLE
Issue #3970: Split Inputs test for EmptyInitializerPadTest

### PR DESCRIPTION
**Pull request by samuel-gu**
Resolved Issue #3970. This pull request is for splitting the input tests for EmptyForInitializerPadTest

### CODE REVIEW

**Comment by rnveach** [comment]
@samuel-gu You do not need to close and create a new PR to fix CI or make changes.
See https://github.com/checkstyle/checkstyle/wiki/PR-rules

**Comment by sabaka** [comment]
@samuel-gu , do not close existent PR next time. Do `git commit --amend` or create new commit and squash all your commits to one when work is finished. We didn't have discussion in previous, but it would be quite hard to follow it if we had.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPad.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
+////////////////////////////////////////////////////////////////////////////////
```
Comment (sabaka) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPad.java]: I don't like this comments.
It was ok for shared inputs, but now they should be fixed to represent current purpose or removed

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPadDefaultConfig.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
+////////////////////////////////////////////////////////////////////////////////
```
Comment (sabaka) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPadDefaultConfig.java]: Same as above

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPad.java (hunk 1 above)
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPad.java]: @sabaka Inputs should be a direct copy or move. We expect no changes to input as this will cause line numbers for violations to change. Plus since this is a new file we will not be able to see what was changed in the input and that it was a valid change.
All changes to inputs will be done in another issue.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPadDefaultConfig.java (hunk 2 above)
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPadDefaultConfig.java]: Same comment as above.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPad.java (hunk 1 above)
Comment (samuel-gu) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPad.java]: @sabaka Should I still make the requested changes?

Comment (sabaka) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPad.java]: @rnveach , ok, makes sense.
@samuel-gu , no, don't do them.

**Comment by sabaka** [comment]
Ok to merge, as for me.
@rnveach , @romani, are you agree?
